assets/translations/en-us.yaml

```diff
@@ -5248,7 +5248,8 @@ typeDescription:
   monitoring.coreos.com.prometheus: A Prometheus server is a Prometheus deployment whose scrape configuration and rules are determined by selected ServiceMonitors, PodMonitors, and PrometheusRules and whose alerts will be sent to all selected Alertmanagers with the custom resource's configuration.
   monitoring.coreos.com.alertmanager: An alert manager is deployment whose configuration will be specified by a secret in the same namespace, which determines which alerts should go to which receiver.
   node: The base Kubernetes Node resource represents a virtual or physical machine which hosts deployments. To manage the machine lifecyle, if available, go to Cluster Management.   
-  catalog.cattle.io.clusterrepo: 'A chart repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo: 'A repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo.local: 'A repository is a Helm repository, {vendor} git based application catalog or a rke template repository. It provides a list of available charts and templates.'
```

Comment:
should rke be capitalised?

Reply:
Strictly speaking the rke template repository is one of the two previous ones, which is offered to the user in a radio button in the create flow.
Maybe something like...

`A repository is either a Helm or {vendor} git based catalog. It provides a list of available Helm charts and templates, and RKE templates.`

for non-local...
`A repository is either a Helm or {vendor} git based catalog. It provides a list of available charts and templates.`